.gitignore

```diff
@@ -21,3 +21,4 @@
 ###< symfony/webpack-encore-pack ###
 
 /yarn-error.log
+.idea
```

Comment:
This should actually belong in your local .gitignore, probably globally? Does not belong in the repo.